Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,62 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
+        public static void Main()
+            {
+                PersonDetails();
+            }
+
+        public static void PersonDetails()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this method's recursion.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845680)